### PR TITLE
data category & example to array.stoplight cleaned

### PR DESCRIPTION
**Pull request by Halkenhaeusser**
Stoplight.io (the editor I am using) re-ordered some lines and removed some clutter/unnecessary lines. E.g. all additionalProperties = true lines were removed because the default for this option is true.

### CODE REVIEW

File: tilt-schema.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
+  "x-stoplight": {
```
Comment (eliasgruenewald) [inline tilt-schema.json:3]: to be removed

File: tilt-schema.json [hunk 2]
```diff
@@ -1373,10 +1345,21 @@
                   }
                 }
               }
-            }
+            },
```
Comment (eliasgruenewald) [inline tilt-schema.json:1348]: ist das hier tatsächlich neu oder nur verschoben? pls double-check

Comment (eliasgruenewald) [reply tilt-schema.json:1348]: (also der ganze Block ab line 1348)

File: tilt-schema.json [hunk 3]
```diff
@@ -746,12 +730,15 @@
               },
               "category": {
                 "$id": "#/properties/dataDisclosed/items/anyOf/0/properties/category",
-                "type": "string",
+                "type": "array",
                 "title": "Category",
-                "description": "The data (category) the data disclosed is referred to.",
+                "description": "The data (category) the data disclosed are referred to.",
```
Comment (eliasgruenewald) [inline tilt-schema.json:735]: In die description vielleicht noch sowas wie "can be multiple".

Comment (Halkenhaeusser) [reply tilt-schema.json]: done

File: tilt-schema.json (hunk 1 above)
Comment (Halkenhaeusser) [reply tilt-schema.json]: done

**Pull request merged**